include/constants.hpp

```diff
@@ -16,6 +16,8 @@ static constexpr uint8_t KW_VPD_PAIR_START_TAG = 0x84;
 static constexpr uint8_t ALT_KW_VPD_PAIR_START_TAG = 0x90;
 static constexpr uint8_t KW_VPD_END_TAG = 0x78;
 static constexpr uint8_t KW_VAL_PAIR_END_TAG = 0x79;
+static constexpr uint8_t AMM_ENABLED_IN_VPD = 2;
+static constexpr uint8_t AMM_DISABLED_IN_VPD = 1;
```

Comment:
AMM_ENABLED_IN_VPD - says AMM is already enabled in VPD. which doesn't give the right sense here. 
can we have it like -
ENABLE_AMM = 2 ;
DISABLE_AMM = 1;

Reply:
It is backing up the data the BIOS.
In BIOS it is already enabled. That "Enabled" is saved into VPD that is why names as "AMM_ENABLED  + IN_VPD".

Reply:
It is open for suggestion though.

Reply:
Either AMM_ENABLED or ENABLED_AMM does not matter to me as long as it have in VPD as it is no the same meaning outside VPD context

Reply:
btw, I didn't mean to swap the words..

Reply:
let leave it for the time being